### ISSUE TITLE
Empty / corrupted epub and pdf for a fic with large number of chapters

### ISSUE THREAD

**Issue by brilliantnut** [waylaidwanderer/FicSave#43]
Hi,

I found an issue downloading both epub and pdf for a fic with a large number of chapters (~87).

The fic in question is https://www.fanfiction.net/s/9750991/1/Angry-Harry-and-the-Seven
I tried the download as an epub and pdf on my MacBookPro using Chrome, and the download happened rather quickly, for such a large fic. However, the epub didn't open in iBooks - saying that the ebook is corrupted, and the PDF had the chapter list and just empty pages for each chapter..

On my mobile (Note 4), downloading an epub, and opening in Aldiko, I found a similar behaviour - I can see the Table of Contents, but the pages are empty after that.

Cheers,
brilliantnut

**Comment by ENDER3000** [comment]
Interesting. Just tried PDF (not EPUB) and you're right, only chapters, no text. What is odd is that I tried it with this fic: https://www.fanfiction.net/s/5218118/1/Yet-again-with-a-little-extra-help and it worked fine. Took a bit, few minutes, but a 6MB PDF file came out. I'm now trying with this fic: https://www.fanfiction.net/s/7920322/41/Mass-Effect-Human-Revolution in which another poster had issues with, with regards to download time (EPUB would not download, https://github.com/waylaidwanderer/FicSave/issues/49) but I got PDF and EPUB to work. 

edit: Was able to DL the Mass Effect with with PDF just fine. a 3.4MB PDF with text. This is odd :confounded:
